Controllers/ApplicationController.cs

```diff
@@ -134,8 +134,11 @@ private static Member InitializeMember(TokenClient tokenClient)
             // The Token UI shows this (and the alias) to the user when requesting access.
             member.SetProfile(new Profile
             {
-                DisplayNameFirst = "PFM Demo"
+                DisplayNameFirst = "Demo",
+                DisplayNameLast = "PFM"
             });
+            byte[] pict = System.IO.File.ReadAllBytes("Content/favicon.png");
```

Comment:
Why favicon? it should be a logo

Reply:
Gianluca told me to use it (from merchant-demo) - it is called favicon.ico there. I already used it in merchant and pfm sample-java.
Do you want me to rename it?